tests/search/attributebitvectors/attributebitvectors.rb

```diff
@@ -0,0 +1,148 @@
+# Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+require 'search_test'
+
+class AttributeBitVectors < SearchTest
+
+  def setup
+    set_owner('toregge')
+    set_description("Test use of attribute posting list bitvectors")
+    @num_docs = 4096
+    @rank_profiles = [ 'unranked', 'default', 'onlya', 'onlyaf' ]
+  end
+
+  def get_app
+    sc = SearchCluster.new('attributebitvectors')
+    sc.sd(selfdir + '/attributebitvectors.sd').threads_per_search(1)
+    app = SearchApp.new.cluster(sc)
+  end
+
+  def word_data
+    'echo "50 rare 200 common 3846 stop"'
+  end
+
+  def doc_template
+    '{ "put": "id:ns:attributebitvectors::$seq()", "fields": { "a": { "$words(1)" : 200 } } }'
+  end
+
+  def feed_data
+    command = DataGenerator.new.feed_command(template: doc_template, count: @num_docs, data: word_data)
```

Comment:
It might not be obvious for the person reading this test what is the result of feeding this dataset. Please add a comment on what is the occurrences of 'rare', 'common', 'stop', and what we can expect regarding the posting lists created for these terms in the search core.

Reply:
Done.